code/WorkInProgress/nadir_antenna.dm

```diff
@@ -346,14 +373,16 @@ var/global/obj/machinery/communications_dish/transception/transception_array
 			return
 		if(cargo_index || manual_receive)
 			var/obj/inbound_target
+			var/was_manual
 			if(manual_receive)
 				inbound_target = manual_receive
+				was_manual = TRUE
 			else if(shippingmarket.pending_crates[cargo_index])
 				inbound_target = shippingmarket.pending_crates[cargo_index]
 			else
 				return
 			if(inbound_target)
-				receive_a_thing(netnum,inbound_target)
+				receive_a_thing(netnum,inbound_target,was_manual)
```

Comment:
```suggestion
				receive_a_thing(netnum,inbound_target, was_manual)
```

Reply:
why can't you just pass `manual_receive`

Reply:
yes so why not just replace manual receive instead of making was manual

Reply:
Because you still need to pass the object through??????

Reply:
attempt_transceive can be prompted in two ways: by a cargo pad (which passes in an index, which is then used to determine inbound_target), or by a directly received object (which passes in that object and makes it the inbound_target). I do not understand a way in which the object given to receive_a_thing could clue it into what sort of reception is taking place.

Reply:
Clarified externally that yes, you just pass in the reference instead of a boolean to not create another variable. Did the thing in 679d391